sklearn/preprocessing/_csr_polynomial_expansion.pyx

```diff
@@ -1,20 +1,46 @@
 # Author: Andrew nystrom <awnystrom@gmail.com>
 
```

Comment:
I think you can add yourself as an author here (I am not suggesting because I do not know which email you would like to use).